docs/docs/user/subscribing.md

```diff
@@ -105,6 +105,19 @@ failure the rescue team has one hour to fix the problem before any event will be
 
 By default inflight TTL is set to 3600 seconds (an hour) and retry backoff is set to 100ms.
 
+## Adaptive backoff
+
+The client is able to signal it can't handle the message at the moment and Hermes Consumer will retry delivery after
+minimum of given delay.
+
+The endpoint can return **Retry-After** header, with the amount of seconds to backoff, combined with status **503**.
```

Comment:
Actually, it's not a maximum in which we'll retry. Just to keep it simple: the TTL will still expire messages regardless of the Retry-After logic. It's up to the user to ensure the Retry-After delay is less than Inflight TTL. I'll update the docs.